src/feature/articles/constants.js

```diff
@@ -2,3 +2,10 @@ export const CREATE_ARTICLE_SUCCESS = 'CREATE_ARTICLE_SUCCESS';
 export const CREATE_ARTICLE_FAIL = 'CREATE_ARTICLE_FAIL';
 export const GET_ARTICLES_SUCCESS = 'GET_ARTICLES_SUCCESS';
 export const GET_ARTICLES_FAIL = 'GET_ARTICLES_FAIL';
+export const GET_SINGLE_ARTICLE_SUCCESS = 'GET_SINGLE_ARTICLE_SUCCESS';
+export const GET_SINGLE_ARTICLE_FAIL = 'GET_SINGLE_ARTICLE_FAIL';
+export const LIKE_ARTICLE_SUCCESS = 'GET_SINGLE_ARTICLE_SUCCESS';
+export const DISLIKE_ARTICLE_FAIL = 'GET_SINGLE_ARTICLE_FAIL';
+export const LIKE_ARTICLE_FAIL = 'GET_SINGLE_ARTICLE_SUCCESS';
+export const DISLIKE_ARTICLE_SUCCESS = 'GET_SINGLE_ARTICLE_FAIL';
```

Comment:
@bdushimi this was because I was working under the feature getting a single article which has been combined with like and dislike article so there were a lot of bugs with getting single article which was resolved on it's feature and made me pull again to get the real working codes, which is why you see some changes on like dislike article and getting single article files.